src/test/test.js

```diff
@@ -0,0 +1,30 @@
+'use strict';
```

Comment:
Do we actually need this JS counterpart of the TS file?

Reply:
That's a mistake, this file should not be a part of the PR altogether, will remove it.